package.json

```diff
@@ -11,7 +11,7 @@
     "@monaco-editor/react": "^4.2.1",
     "@sentry/react": "^6.10.0",
     "@sentry/tracing": "^6.10.0",
-    "@supabase/supabase-js": "^1.15.1",
+    "@supabase/supabase-js": "^1.22.5",
```

Comment:
comes with a fix for accessing `localStorage` during initialization of the module which has been throwing in incognito mode (and probably under some privacy settings as well). Thanks for the upgrade I could finally remove our patch for this issue